### ISSUE TITLE
Add term flared proximal tibial metaphysis

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10500]
**Preferred term label:**
Flared proximal tibial metaphysis

**Synonyms**

**Definition (free text, please give PubMed ID)**
The presence of a splayed (i.e.,flared) metaphyseal segment of the proximal tibia.
Smithson S. (2019). Jürgen W. Spranger, Paula W. Brill, Christine Hall, Gen Nishimura, Andrea Superti-Furga, and Sheila Unger: Bone dysplasias: an atlas of genetic disorders of skeletal development: Fourth Edition, ISBN: 9780190626655. Human Genetics, 138(7), 787. https://doi-org.mu.idm.oclc.org/10.1007/s00439-019-02038-0

**Parent term (use hpo.jax.org/app)**
Flared lower limb metaphysis HP:0030307

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
For example, Cartilage-hair hypoplasia (omim250250) PMID22987807
SMED aggrecan type (omim612813), Smithson S. (2019). Jürgen W. Spranger, Paula W. Brill, Christine Hall, Gen Nishimura, Andrea Superti-Furga, and Sheila Unger: Bone dysplasias: an atlas of genetic disorders of skeletal development: Fourth Edition, ISBN: 9780190626655. Human Genetics, 138(7), 787. https://doi-org.mu.idm.oclc.org/10.1007/s00439-019-02038-0

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@lisannesw 
Adding new term: 
- ID: HP:6000951
- label: Flared proximal tibial metaphysis
- parents: [Flared lower limb metaphysis (HP:0030307)](https://hpo.jax.org/app/browse/term/HP:0030307)
- pmid: 
- def: The presence of a splayed (i.e.,flared) metaphyseal segment of the proximal tibia.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10500
- synonyms:
